### ISSUE TITLE
Fullscreen mode does not increase image display size

### ISSUE THREAD

**Issue by Globulle** [mhkeller/pulp2#6]
This issue is visible directly on [the demo](https://mhkeller.github.io/pulp2/) : when I switch to fullscreen mode, the browser toolbar disappears, but the image is not bigger (there is some white space below).

I would expect to have the comics pages occupying the whole height of my screen (as it is the case on the [aljazeera's project](http://projects.aljazeera.com/2014/terms-of-service/)).

I assume this is not due to image resolution since it is 1500 px-high, which is higher than the height of my screen (1080px).

Thanks for your help.

**Comment by Globulle** [comment]
Update : the problem is more visible if your browser does not occupies your whole screen before switching to fullscreen mode.

As far as I know the problem occurs with the following configs:
* Firefox 51.0.1 (64 bits) under Ubuntu 16.04
* Firefox 51 under Andergos (Arch-based distrib) and Gnome 3
* Firefox 45.3.0 with Debian Jessie
* Firefox 50.1.0 with WIndows 8.1 
* Chrome 41.0 with Windows 8.1

It seem to not occur with Chromium 56.

**Comment by mhkeller** [comment]
Ya I'm seeing it on Firefox 51.0.1 and 52 on MacOS. It must not be firing a resize event. Looking into it.

**Comment by mhkeller** [comment]
@Globulle it should be fixed via these two commits. I also updated some of the code style and got rid of some unused variables

* https://github.com/mhkeller/pulp2/commit/a4384bcb74686d60cb330a86068f84b6e6655ffe
* https://github.com/mhkeller/pulp2/commit/a0a971253b117cbac673c443b8bda0f945b00f69

The issue was [this line](https://github.com/mhkeller/pulp2/blob/609a96dac0602bed2301fe61150481792f302f8d/src/js/main.js#L282), which was added to support the hover functionality, which wasn't in the Terms of Service version so hence why it didn't have that bug. The issue was that I had to add another mask div to support the desired hover zoom behavior. That width was limiting the expansion when the screen got bigger. The fix is a little ugly in that I stash the original width onto `settings` and use this to set the width both on resize and on page change.

Let me know if this fixes it in your environment and I'll close.
